### PR TITLE
Bump version to 0.6.0b8, Added `cp37-abi3-win32` build

### PR DESCRIPTION
**Pull request by T-256**
part of #207

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -9,24 +9,43 @@ on:
...
-          platforms: all
-      # macOS make is too old
-      - if: runner.os == 'macOS'
-        run: brew install make
+          config-file: "{package}/ci-win32.toml"
+      - uses: actions/upload-artifact@v3
+        with:
+          path: ./wheelhouse/*.whl
+  windows-x86_64:
+    name: build windows wheels and test
```
Comment (T-256) [inline .github/workflows/test.yaml:28]: perhaps, we could use matrix here. but I don't touch it as it is working.

File: ci-win32.toml
Comment (yifeikong) [inline ci-win32.toml:1]: Could you please consolidate this file to pyproject.toml? It's easier to manage.

Comment (T-256) [reply ci-win32.toml:1]: It's a parameter for ci: https://github.com/yifeikong/curl_cffi/pull/219/files#diff-245392b692a50c38ecab4381b118862db514035c10983f3bd4f4b7f1f4be4692R23

Do you know better way to consolidate it to pyproject.toml and pass it as argument to ci?

File: preprocess/download_so.py [hunk 2]
```diff
@@ -28,21 +21,35 @@ def reporthook(blocknum, blocksize, totalsize):
...
-filename = "./curl-impersonate.tar.gz"
-if url:
-    print(f"Download libcurl-impersonate-chrome from {url}")
-    urlretrieve(url, filename, (None if os.getenv("GITHUB_ACTIONS") else reporthook))
-    shutil.unpack_archive(filename, LIBDIR)
-    if uname.system == "Windows":
-        shutil.copy2(f"{LIBDIR}/libcurl.dll", "curl_cffi")
+    libdir = "/usr/local/lib"
+
+if len(sys.argv) == 3 and sys.argv[2] == '1':  # i686 32-bit flag
```
Comment (yifeikong) [inline preprocess/download_so.py:35]: Checking sys.argv with more than one arguments is kind of code smell, but let's leave it as is for now, we should make the names more consistent in curl-impersonate release files and move the code in download_so.py to Makefile in the future.

File: .github/workflows/test.yaml (hunk 1 above)
Comment (T-256) [reply .github/workflows/test.yaml:28]: Edit by maintainers is on for this PR, feel free to apply matrix on this change.

File: ci-win32.toml
Comment (yifeikong) [reply ci-win32.toml:1]: use the postfix

```
[tool.cibuildwheel.macos]
```

Comment (T-256) [reply ci-win32.toml:1]: This file for Architecture isolation.
Edit by maintainers is on for this PR, feel free to apply what you mean on this change.

File: preprocess/download_so.py (hunk 2 above)
Comment (T-256) [reply preprocess/download_so.py:35]: Correct, tracked in #197

**Comment by yifeikong** [comment]
It's trickier than I thought. `cibuildwheel` only runs the 32bit python after `before-all`, but `before-all` always use the system's python, which is 64bit.

**Comment by T-256** [comment]
> It's trickier than I thought. `cibuildwheel` only runs the 32bit python after `before-all`, but `before-all` always use the system's python, which is 64bit.

True, this is exactly what I mean by `Architecture isolation` at above.
Do you mind revert to main commit of this PR then merge?

**Comment by yifeikong** [comment]
`setup.py` now downloads libcurl-impersonate automatically, if your PR #192 includes the patched header files, users will be able to install from a sdist, which means they should work for platforms like raspberry pi and termux.

File: pyproject.toml [hunk 3]
```diff
@@ -84,20 +84,33 @@ build-backend = "setuptools.build_meta"
...
+# configure cibuildwheel to build native archs ('auto'), and some emulated ones
+[tool.cibuildwheel.linux]
+archs = ["auto", "aarch64"]
+
 [tool.cibuildwheel.macos]
 before-all = "gmake preprocess"
 
-
 [tool.isort]
 profile = "black"
```
Comment (T-256) [inline pyproject.toml:109]: could we drop `isort` in favor of `ruff`?

File: setup.py [hunk 4]
```diff
@@ -13,8 +23,68 @@ def get_tag(self):
         return python, abi, plat
 
 
+def download_so():
+    uname = platform.uname()
+    machine = uname.machine
+
+    # do not download if target platfrom dll not found
```
Comment (T-256) [inline setup.py:30]: move this comment to proper place.

**Review by T-256 — COMMENTED** [review]
LGTM, Thanks!

File: pyproject.toml (hunk 3 above)
Comment (yifeikong) [reply pyproject.toml:109]: Yes, I'm using ruff actually.

**Comment by T-256** [comment]
> if your PR #192 includes the patched header files, users will be able to install from a sdist, which means they should work for platforms like raspberry pi and termux

I think in #192 sdist includes `ffi` and `include` folder, but I'm not sure they are patched or not. I'll check now.

File: setup.py (hunk 4 above)
Comment (yifeikong) [reply setup.py:30]: Feel free to delete it in your future PR.

**Pull request merged**